tests/Io/FileSystemTest.php

```diff
@@ -42,6 +42,130 @@ public function tearDown()
         $this->exec('rm -rf ' . $this->tmp);
     }
 
+    /**
+     * @expectedException        Brick\Std\Io\IoException
+     * @expectedExceptionMessage Error getting real path of invalid_path, check that the path exists
+     */
+    public function testGetRealPathWithInvalidPath()
+    {
+        FileSystem::getRealPath('invalid_path');
+    }
+
+    public function testGetRealPath()
+    {
+        $filePath = __DIR__ . '/../../composer.json';
+        $expectedRealPath = realpath($filePath);
+
+        $this->assertSame($expectedRealPath, FileSystem::getRealPath($filePath));
+    }
+
+    public function testWriteWithAppendFlag()
+    {
+        $this->assertSame(4, FileSystem::write('./temp_file', 'data', true));
+    }
+
+    public function testWriteWithLockFlag()
+    {
+        $this->assertSame(0, FileSystem::write('./temp_lock_file', '', false, true));
+    }
+
+    public function testReadWithMaxLength()
+    {
+        FileSystem::write('./temp_lock_file', 'data');
+
+        $this->assertSame('dat', FileSystem::read('./temp_lock_file', 0, 3));
+    }
+
+    /**
+     * @expectedException        Brick\Std\Io\IoException
+     * @expectedExceptionMessage Error copying ./temp_lock_file to /temp_lock_file
+     */
+    public function testCopyShouldReturnIOException()
+    {
+        FileSystem::write('./temp_lock_file', 'data');        
+        FileSystem::copy('./temp_lock_file', '/temp_lock_file');
+    }
+
+    /**
+     * @expectedException        Brick\Std\Io\IoException
+     * @expectedExceptionMessage Error moving ./temp_lock_file to /temp_lock_file
+     */
+    public function testMoveShouldReturnIOException()
+    {
+        FileSystem::write('./temp_lock_file', 'data');        
+        FileSystem::move('./temp_lock_file', '/temp_lock_file');
+    }
+
+    /**
+     * @expectedException        Brick\Std\Io\IoException
+     * @expectedExceptionMessage Error deleting /temp_lock_file
+     */
+    public function testDeleteShouldReturnIOException()
+    {
+        FileSystem::delete('/temp_lock_file');
+    }
+
+    /**
+     * @expectedException        Brick\Std\Io\IoException
+     * @expectedExceptionMessage Error creating directory /temp_directory
+     */
+    public function testCreateDirectoryShouldReturnIOException()
+    {
+        FileSystem::createDirectory('/temp_directory');
+    }
+
+    /**
+     * @expectedException        Brick\Std\Io\IoException
+     * @expectedExceptionMessage Error creating directories /temp_directory
+     */
+    public function testCreateDirectoriesShouldReturnIOException()
+    {
+        FileSystem::createDirectories('/temp_directory');
+    }
+
+    public function testCreateDirectoriesShouldReturnNull()
+    {
+        FileSystem::createDirectories('./temp_directory');
+
+        $this->assertNull(FileSystem::createDirectories('./temp_directory'));
```

Comment:
Redundant line, and useless assertNull() call.

Reply:
This assertion is tested this following line:
```
if (self::isDirectory($path)) {
   return;
}
```

Reply:
Ah OK, the naming does not make it clear. What about `testCreateDirectoryTwice`?
And please remove this `assertNull()` ;)

Reply:
If I don't use the ```assertNull``` assertion, what's proper asserions for this test?

Reply:
Now it presents the Risky test:

```
1) Brick\Std\Tests\Io\FileSystemTest::testCreateDirectoryTwice
This test did not perform any assertions
```

Reply:
I'll take care of this one :)